modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-bonded-sriov-network_{context}"]
+= Configure sysctl on a bonded SR-IOV network 
+
+As a cluster administrator, set the interface specific sysctls on a bonded interface created from two SR-IOV interfaces by adding the tuning configuration to the optional `metaPlugins` parameter of the bond network attachment definition.
+
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` custom resources that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+To change all interface-level network sysctls create an additional SR-IOV network attachment with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create a bond network attachment definition and insert the `metaPlugins` configuration, as in the following example CR. Save the YAML as the file `sriov-network-interface-sysctl-all.yaml`.
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
It looks like it's invalid yaml, i've tried to create it and failed.

Reply:
I've used my bond yaml. You can see it below:

```
apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: bond-net1
  namespace: sysctl-tuning-test
spec:
  config: '{
  "cniVersion":"0.4.0",
  "name":"bound-net",
  "plugins":[
    {
      "type":"bond",
      "ifname":"bond0",
      "mode": "active-backup",
      "failOverMac": 1,
      "linksInContainer": true,
      "miimon": "100",
      "links": [
        {"name": "net1"},
        {"name": "net2"}
      ],
      "ipam":{
        "type":"static"
      }
    },
    {
      "type":"tuning",
      "capabilities":{
        "mac":true
      },
      "sysctl":{
        "net.ipv4.conf.IFNAME.accept_redirects": "0",
        "net.ipv4.conf.IFNAME.accept_source_route": "0",
        "net.ipv4.conf.IFNAME.disable_policy": "1",
        "net.ipv4.conf.IFNAME.secure_redirects": "0",
        "net.ipv4.conf.IFNAME.send_redirects": "0",
        "net.ipv6.conf.IFNAME.accept_redirects": "0",
        "net.ipv6.conf.IFNAME.accept_source_route": "1",
        "net.ipv6.neigh.IFNAME.base_reachable_time_ms": "20000",
        "net.ipv6.neigh.IFNAME.retrans_time_ms": "2000"
      }
    }
  ]
}'

```

Reply:
used this example in docs to illustrate how to configure